### ISSUE TITLE
No results in Chrome under Linux

### ISSUE THREAD

**Issue by Farnsi** [thetarnav/solid-devtools#229]
Downloaded your Stackblitz example and run it localy, but Solid tab did not show any component.

Plugin output:

```
Solid detected

Devtools client detected

Extension: 0.22.0.0
Client: 0.26.0
Expected client: 0.26.0
```

Solid Tab shows only: 
```
Solid Developer Tools
#0.22.0.0_0.26.0/0.26.0
Search | Components | Owners | DOM
```

Console

```
🚧 solid-devtools is in early development! 🚧
Please report any bugs to https://github.com/thetarnav/solid-devtools/issues
```

Browser Chrome Version 111.0.5563.146 

Feature Request: use browser dev tools theme colors for dark view.

**Comment by Takkun053** [comment]
The same thing happened with my Linux.

Below are the results of the Solid tab verification.

[devtools-1680494033950.log](https://github.com/thetarnav/solid-devtools/files/11134577/devtools-1680494033950.log)

**Comment by StitiFatah** [comment]
Same on Arch (so everything up to date), tested on Chromium 113.0.5672.92 , Brave 113.1.51.114 ,  Google Chrome 113.0.5672.92, Google Chrome unstable 115.0.5762.4 dev

**Comment by aryzing** [comment]
Seems this fixed? At least on Ubuntu 22.04 LTS w/ Chrome it's working.
